components/push/storage/src/db.rs

```diff
@@ -112,22 +110,24 @@ impl Storage for PushDb {
         let affected_rows = self.execute_named(
             "DELETE FROM push_record
              WHERE uaid = :uaid AND channel_id = :chid",
-            &[(":uaid", &uaid as &ToSql),
-              (":chid", &chid as &ToSql)],
+            &[(":uaid", &uaid as &ToSql), (":chid", &chid as &ToSql)],
```

Comment:
(This applies to a few places) You should be able to just do `&uaid` instead of `&uaid as &ToSql` (logins was written a long time ago, and I only I cleaned it up in the first part of #708).

If you did need to do &ToSql, we generally do `&dyn ToSql` to indicate that something is a trait object, but it's not huge.